deps/chakrashim/src/v8message.cc

```diff
@@ -28,8 +28,18 @@ Local<String> Message::Get() const {
 }
 
 MaybeLocal<String> Message::GetSourceLine(Local<Context> context) const {
-  // CHAKRA-TODO: Figure out how to transmit this info...?
-  return Local<String>();
+  JsPropertyIdRef source = jsrt::IsolateShim::GetCurrent()
+    ->GetCachedPropertyIdRef(jsrt::CachedPropertyIdRef::source);
```

Comment:
yeah, i agree. Please follow the indentation we use in other files.

---
In reply to: [118531320](https://github.com/nodejs/node-chakracore/pull/263#discussion_r118531320) [](ancestors = 118531320)